cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,133 @@
+    <label for="partition-select" class="group-select-title"><%- gettext('Restrict access to:') %>
+        <select class="user-partition-select" id="partition-select">
+            <option value="-1" selected ="selected"><%- gettext('Select a group type') %></option>
+            <% for (var i=0; i < selectablePartitions.length; i++) { %>
+                <option id="<%- selectablePartitions[i].id %>-option" value="<%- selectablePartitions[i].id %>"><%- selectablePartitions[i].name %></option>
+            <% } %>
+        </select>
+    </label>
+    <br>
+    <% if (selectablePartitions.length > 0) { %>
+        <div class="user-partition-group-checkboxes">
+            <% for (var i=0; i < selectablePartitions.length; i++) { %>
+                <div id="<%- selectablePartitions[i].id %>-checkboxes">
+                    <div class="partition-group-directions">
+                        <%- gettext('Select one or more groups:') %>
+                        <% for (var j = 0; j < selectablePartitions[i].groups.length; j++) { %>
+                            <div class="field partition-group-control">
+                                <input type="checkbox" id="content-group-<%- selectablePartitions[i].groups[j].id %>" value="<%- selectablePartitions[i].groups[j].id %>" class="input input-checkbox"
+                                    <% // Checkbox is checked if the partition exists and the specific partition's group
+                                       // is in the xblock's group access, or if the group has been deleted. %>
+                                    <% if ((groupAccess[selectablePartitions[i].id]
+                                            && groupAccess[selectablePartitions[i].id].indexOf(selectablePartitions[i].groups[j].id) !== -1)
+                                            || selectablePartitions[i].groups[j].deleted) { %> checked="checked" <% } %> />
+                                <% if (selectablePartitions[i].groups[j].deleted) { %>
+                                    <label for="content-group-<%- selectablePartitions[i].groups[j].id %>" class="label deleted">
+                                        <%- gettext('Deleted Group') %>
+                                        <span class="deleted-group-message"><%- gettext('This group no longer exists. Choose another group or do not restrict access to this unit.') %></span>
+                                <% } else { %>
+                                    <label for="content-group-<%- selectablePartitions[i].groups[j].id %>" class="label">
+                                        <%- selectablePartitions[i].groups[j].name %>
+                                    </label>
+                                <% } %>
+                            </div>
+                        <% } %>
+                    </div>
+                </div>
+            <% } %>
+        </div>
+    <% } else { %>
+        <%- gettext('You have not set up any content groups yet.') %>
```

Comment:
As I mentioned before, I don't think this is the right message to show in this case. Please work with @roderickmorales.